src/spid-cache.js

```diff
@@ -12,36 +12,25 @@
     }
```

Comment:
Maybe the `spid-cache.js` should actually have some sort of expiration mechanism?